app/src/main/java/com/github/multimatum_team/multimatum/repository/FirebaseDeadlineRepository.kt

```diff
@@ -48,7 +50,22 @@ class FirebaseDeadlineRepository @Inject constructor() : DeadlineRepository {
             .ofEpochMilli(milliseconds)
             .atZone(ZoneId.systemDefault())
             .toLocalDateTime()
-        return Deadline(title, state, date)
+        var description = ""
+        if (deadlineSnapshot["description"] != null) {
+            description = deadlineSnapshot["description"] as String
+        }
```

Comment:
I think it could be simplified:
```Kotlin
val snapshot = deadlineSnapshot["description"]
val description = if (snapshot != null) (snapshot as String) else ""
```
And Android Studio will possibly advise something better using the Elvis operator, or something similar.

Reply:
ok

Reply:
ça fait crash le programme si je fait ça donc je vais laisser comme avant